test/classes/initializers/initequals/wrongType.good

```diff
@@ -1,2 +1,4 @@
-wrongType.chpl:6: In function 'init=':
-wrongType.chpl:7: error: initializing field 'T' with type '2*int(64)', expecting 'int(64)'
+wrongType.chpl:6: In method 'init=':
+wrongType.chpl:7: error: Cannot replace an instantiated field with another type
+wrongType.chpl:7: note: field T has type int(64) but is set to 2*int(64)
```

Comment:
I somehow feel more OK with types not being in quotes, but if you want to do that, that's also fine